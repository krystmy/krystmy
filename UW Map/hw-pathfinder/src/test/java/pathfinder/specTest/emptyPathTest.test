#test finding a path to itself

CreateGraph graph

AddNode graph A

FindPath graph A A
