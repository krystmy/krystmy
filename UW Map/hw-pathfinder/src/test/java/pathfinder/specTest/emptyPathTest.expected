#test finding a path to itself

created graph graph

added node A to graph

path from A to A:
total cost: 0.000